src/pages/AddRecord/AddRecordCategory.tsx

```diff
@@ -40,11 +49,47 @@ function AddRecordCategory({
       { title: '내편이 되어주세요', choosed: false, id: 10, iconSrc: MySide },
     ],
   }
-
   const [categoryState, setCategoryState] = useState<CategoryType>(categoryData)
   const [formData, setFormData] = useRecoilState(formDataAtom)
 
+  useEffect(() => {
+    const getData = async () => {
+      const { data } = await getCategory()
+      setCategoryState(makeCategoryData(data))
+      console.log(data)
+    }
+    getData()
+  }, [])
+
+  const makeCategoryData = (data: CategoryDatas) => {
+    const categoryData: CategoryType = {
+      [TEXT_DETAILS.CELEBRATION]: data[1].subcategories.map(
+        (category: BigCategory, index: number) => {
+          return {
+            title: category.name,
+            choosed: index === 0 && true,
```

Comment:
여기 이부분이랑 아래 choosed 부분 변경해주세용

```suggestion
            choosed: index === 0,
```